File: PHP/UberX.php
<?php
require_once('Car.php');

class UberX extends Car{
  public $brand;
  public $model;

  public function __construct(
    string $licence,
    Account $driver,
    string $brand,
    string $model
  ){
    parent::__construct( $licence, $driver);
    $this->brand = $brand;
    $this->model = $model;
  }
}

File: Java/UberX.java
package Java;

public class UberX extends Car {
  String brand;
  String model;

  public UberX(
    String licence,
    Driver driver,
    String brand,
    String model
  ){
    super( licence, driver);
    this.brand = brand;
    this.model = model;
  }
}
